node-operators/networks/collators/orbiter.md

```diff
@@ -9,140 +9,161 @@ description: Learn about the Moonbeam Orbiter Program for collators including th
+    {{ networks.moonbase.orbiter.bond }} DEV
     ```
 
 ## Rewards {: #rewards }
 
-Rewards for Orbiters will be split between the other Orbiters assigned to the same Program Collator. The maximum Orbiters per Program Collator is described in the [configuration section](#configuration). In the case of Moonriver it’s {{ networks.moonriver.orbiter.max_orbiters_per_collator }}, so the rewards will be approximately 1/{{ networks.moonriver.orbiter.max_orbiters_per_collator }} of a collator’s rewards. The blocks produced by each Orbiter while active are tracked, and rewards are proportionally distributed.
+Rewards for orbiters will be split between the other orbiters assigned to the same orbiter pool. The maximum orbiters per orbiter pool is described in the [configuration section](#configuration). In the case of Moonriver it’s {{ networks.moonriver.orbiter.max_orbiters_per_collator }}, so the rewards will be approximately 1/{{ networks.moonriver.orbiter.max_orbiters_per_collator }} of a collator’s rewards. The blocks produced by each orbiter while active are tracked, and rewards are proportionally distributed.
 
 ## Performance Metrics {: #performance-metrics }
 
-Each Orbiter’s performance will be assessed over a period of time to determine they are active and producing blocks, and if their performance is within a range of all other program collators. Orbiters are expected to run top tier hardware to stay within range. For more information on hardware requirements, please check out the [Collator Requirements page](/node-operators/networks/collators/requirements/){target=_blank}. 
+Each orbiter’s performance will be assessed over a period of time to determine they are active and producing blocks, and if their performance is within a range of all other orbiter pool collators. Orbiters are expected to run top tier hardware to stay within range. For more information on hardware requirements, please check out the [Collator Requirements page](/node-operators/networks/collators/requirements/){target=_blank}. 
 
-Metrics will be assessed over 7 day periods. The performance metrics are as follows:
+Metrics will be assessed over seven day periods. The performance metrics are as follows:
 
-- Orbiter has produced a block within the last 3 rounds they were active 
-- Orbiter’s block production is within 2 standard deviation of the 7 day program mean
-- Orbiter’s transactions per block is within 2 standard deviation of the 7 day program mean 
-- Orbiter’s block weight is within 2 standard deviation of the 7 day program mean
+- Orbiter has produced a block within the last three rounds they were active 
+- Orbiter’s block production is within two standard deviation of the seven day program mean
+- Orbiter’s transactions per block is within two standard deviation of the seven day program mean 
+- Orbiter’s block weight is within two standard deviation of the seven day program mean
 
 !!! note
     These factors are subject to change during this trial period.
```

Comment:
let's get rid of trial period, something like 
These factors are subject to change as the program progresses